<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallFirmwarePage</class>
 <widget class="QWizardPage" name="InstallFirmwarePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QLabel" name="installed_fonts_status">
     <property name="text">
      <string>Installed</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="download_font_package_button">
     <property name="text">
      <string>Download Fonts Package</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="download_firmware_button">
     <property name="text">
      <string>Download Firmware</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="page_name">
     <property name="text">
      <string>Install Firmware.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="installed_firmware_status">
     <property name="text">
      <string>Installed</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="install_firmware_file_button">
     <property name="text">
      <string>Install Firmware File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
